chromium_src/chrome/common/channel_info_posix.cc

```diff
@@ -18,6 +18,10 @@ std::string GetChannelName() {
   return modifier;
 }
 
+std::string GetChannelSuffixForDataDir() {
+  return std::string();
```

Comment:
this doesn't seem like what we want here, how are we doing this now?

Reply:
`GetChannelImpl` in `channel_info_posix.cc` used to take 2 out params: `std::string* modifier_out` and `std::string* data_dir_suffix_out`. Now it only takes `modifier_out` and the `data_dir_suffix_out` was split into this new separate function. The only place where the new function is called is from `GetDefaultUserDataDirectory` in `chrome_paths_linux.cc`.

We override both `channel_info_posix.cc` and `chrome_paths_linux.cc` files in their entirety (don't include the original) and call our version - `brave::GetChannelImpl` in `brave_channel_info_posix.cc` and our version of `GetDefaultUserDataDirectory` in the `chrome_paths_linux.cc`. 

Didn't seem like we needed to follow the split here since the Chromium comment was that this was only done `if we ever wanted to do release channels for unbranded builds`. 

The compiler was complaining about a missing function implementation, that's why I added it.